### PR TITLE
Update schemes.md with interop notes.

### PR DESCRIPTION
**Pull request by mfoltzgoogle**
Addresses Issue #93:  Add note to schemes.md about custom schemes and interop.

This adds a section noting interop challenges for browsers that support URL schemes other than `https`.

It also updates the existing text to align with the current terminology, adds links to some definitions/specs, and does some general copy-editing.

### CODE REVIEW

**Comment by mfoltzgoogle** [comment]
I think @chrisn and @louaybassbouss have the most context to review this document, so PTAL.

File: schemes.md [hunk 1]
```diff
@@ -57,16 +62,32 @@ technology used by a given screen.
...
-vendor of controlling UA to white/blacklist specific URL schemes.
-
-*TODO:*
-[Issue #93: Custom schemes and interop](https://github.com/webscreens/openscreenprotocol/issues/93)
+The mechanism of using schemes in the OSP allows controlling pages to support
+additional receiver application types.  For example, Android TV could implement
+the OSP receiver for native TV applications in the future and advertise them
+using the scheme `android.` A native Android TV app with a native library that
+acts as a presentation receiver could then be controlled through the
+Presentation Controller API.
```
Comment (chrisn) [inline schemes.md]: I'd like to add another example: "A controlling page could open an HbbTV web application on the OSP receiver and, but using an `hbbtv` URI scheme pass additional parameters that enable the receiving web page to access broadcast-specific functionality"

File: schemes.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
-support Google Cast receivers like Chromecast and Android TV. Let us consider
-the following example as input for discussion:
+display using the 
+[`PresentationRequest`](https://www.w3.org/TR/presentation-api/#interface-presentationrequest)
+interface. The `PresentationRequest` constructor accepts a URL or a sequence of
+URLs of the presentation page as input. While the current Presentation API spec
+defines the behavior of `https` URLs, it does not forbid the usage of other URLs
+with other schemes.
+
+For example the the `cast` scheme is used in the Presentation API tests as
```
[the remark this answers is not in the document]
Comment (mfoltzgoogle) [reply schemes.md]: Done.

File: schemes.md [hunk 3]
```diff
@@ -57,16 +62,32 @@ technology used by a given screen.
...
+acts as a presentation receiver could then be controlled through the
+Presentation Controller API.
+
+## Interoperability
+
+There remain interoperability concerns with supporting non-https schemes through
+the OSP.  A browser may need to support additional features that are not part of
+the core OSP standard to work with a non-https application; this is true of
+`cast:` URLs today, for example.  Browsers that do not have these features may
+believe that a non-https URL as available for presentation (because the
```
[the remark this answers is not in the document]
Comment (mfoltzgoogle) [reply schemes.md]: Good catch, fixed.

File: schemes.md (hunk 1 above)
Comment (mfoltzgoogle) [reply schemes.md]: Done, I lightly edited your suggestion, please let me know if that fits what you expected.

**Comment by mfoltzgoogle** [comment]
I'm going to go ahead and merge this; feel free to add comments at any time @louaybassbouss.

**Pull request merged**